.github/workflows/ci.yml

```diff
@@ -30,3 +30,6 @@ jobs:
 
             - name: eslint
               run: yarn lint
+
+            - name: unit-tests
+              run: yarn test:unit --coverage false
```

Comment:
imho, trying to do 100% coverage is bad practice but following the coverage will show the details of tested and not-tested areas.

it will be nice to see the coverage details in the PR after adding code. we're also doing the exact same for the https://github.com/nextcloud/integration_openproject too.

Reply:
Adding coverage details will be done as a separate task: https://github.com/gherlint/gherlint/issues/20